resources/ts/Components/Integrations/Detail/BillingInfo.tsx

```diff
@@ -56,95 +68,93 @@ export const BillingInfo = ({ id, organisation, subscription }: Props) => {
                         name: e.target.value,
                       })
                     }
-                    disabled={isDisabled}
                   />
                 }
               />
-            </div>
-            <div className="grid grid-cols-3 gap-3 max-md:grid-cols-1">
-              <FormElement
-                label={`${t("details.billing_info.address.street")}`}
-                error={errors["organisation.address.street"]}
-                component={
-                  <Input
-                    type="text"
-                    name="organisation.address.street"
-                    value={data.organisation?.address.street}
-                    onChange={(e) =>
-                      setData("organisation", {
-                        // We know organisation exists
-                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                        ...data.organisation!,
-                        address: {
+              <div className="max-md:flex max-md:flex-col md:grid md:grid-cols-5 gap-3 md:min-w-[40rem]">
+                <FormElement
+                  label={`${t("details.billing_info.address.street")}`}
+                  error={errors["organisation.address.street"]}
+                  className="col-span-2"
+                  component={
+                    <Input
+                      type="text"
+                      name="organisation.address.street"
+                      value={data.organisation?.address.street}
+                      onChange={(e) =>
+                        setData("organisation", {
+                          // We know organisation exists
                           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                          ...data.organisation!.address,
-                          street: e.target.value,
-                        },
-                      })
-                    }
-                    disabled={isDisabled}
-                  />
-                }
-              />
-              <FormElement
-                label={`${t("details.billing_info.address.postcode")}`}
-                error={errors["organisation.address.zip"]}
-                component={
-                  <Input
-                    type="text"
-                    name="organisation.address.zip"
-                    value={data.organisation?.address.zip}
-                    onChange={(e) =>
-                      setData("organisation", {
-                        // We know organisation exists
-                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                        ...data.organisation!,
-                        address: {
+                          ...data.organisation!,
+                          address: {
+                            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                            ...data.organisation!.address,
```

Comment:
```suggestion
                            ...data.organisation.address,
```

If we check `data.organisation` instead of `organisation`, this no null assertion won't be needed

in the code above change
`organisation &&`
to
`data.organisation &&`